content/practices/high-contrast/high-contrast-practice.html

```diff
@@ -0,0 +1,779 @@
+                     class="sample"
+                     style="background-color: #242424"
+                     aria-label="Color is a very dark gray">
+                </div>
+                <span class="color">#242424</span>
+              </td>
+              <td>12.8</td>
+              <td>
+                 <div role="img"
+                     class="sample"
+                     style="background-color: #a1a1a1"
+                     aria-label="Color is a very light gray">
+                </div>
+                <span class="color">#a1a1a1</span>
+              </td>
+              <td>
+                 <div role="img"
+                     class="sample"
+                     style="background-color: #fff"
+                     aria-label="Color is white">
+                </div>
+                <span class="color">#fff</span>
+              </td>
+              <td>4.6</td>
+            </tr>
+            <tr>
+              <th>
+                <code>dark</code>
+              </th>
+              <td>
+                 <div role="img"
+                     class="sample"
+                     style="background-color: #333"
+                     aria-label="Accent color is light gray">
+                </div>
+                <span class="color">#333</span>
+              </td>
+              <td>
+                 <div role="img"
+                     class="sample"
+                     style="background-color: #fff"
+                     aria-label="Color is black">
+                </div>
+                <span class="color">#fff</span>
+              </td>
+              <td>12.6</td>
+              <td>
+                 <div role="img"
+                     class="sample"
+                     style="background-color: #36c"
+                     aria-label="Color is dark blue">
+                </div>
+                <span class="color">#36c</span>
+              </td>
+              <td>
+                 <div role="img"
+                     class="sample"
+                     style="background-color: #fff"
+                     aria-label="Color is white">
+                </div>
+                <span class="color">#fff</span>
+              </td>
+              <td>5.36</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <h4>CSS Media Query Code</h4>
+
+        <pre>
+          <code>
+@media (prefers-color-scheme: dark) {
+  button.color-scheme[role="switch"] {
+    background-color: #333;
+    color: #fff;
+  }
+
+  button.color-scheme[role="switch"] .label {
+    color: #fff;
+  }
+
+  button.color-scheme[role="switch"] svg rect {
+    fill: #36c;
+    stroke: #36c;
+  }
+
+  button.color-scheme[role="switch"] svg circle.off,
+  button.color-scheme[role="switch"] svg circle.on {
+    fill: #fff;
+    stroke: #fff;
+  }
+
+  button.color-scheme[role="switch"]:focus,
+  button.color-scheme[role="switch"]:hover {
+    border-color: #add8e6;
+  }
+}
+          </code>
```

Comment:
Now that [`light-dark()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/light-dark) is widely implemented (only as of May 2024 😅), how about modernizing this code example using `color-scheme` and `light-dark()`? Or adding it as an alternative technique?

Reply:
Sounds like a good idea, want to look like we are keeping up with the times

Reply:
@adampage 
Maybe we could do this for the next publication cycle.

Reply:
Absolutely. 👍🏻